database/datageneration.js

```diff
@@ -1,36 +1,46 @@
 const faker = require('faker');
-const fileRestaurant = require('fs').createWriteStream('database/restaurant.csv');
-const fileMenu = require('fs').createWriteStream('database/menu.csv');
+const fileRestaurant = require('fs').createWriteStream('database/restaurant.txt');
+// const fileRestaurant = require('fs').createWriteStream('database/restaurantShort.txt'); <- use this instead to generate smaller test file
 
 const dishTypes = ['Breakfast', 'Lunch', 'Dinner', 'Brunch', 'Bar', 'Dessert', 'Cheese', 'Craft Cocktails & Libations', 'Afternoon Bites', 'After Dinner', 'Wines by the Glass'];
 const subTypes = ['Mains', 'Starters', 'Sides', 'Raw Bar', 'Snacks', 'Appetizers', 'Classics', 'Recommendations', 'Entrees', 'Small Plates', 'Soups & Salads'];
 
 const generateRestaurants = async () => {
-  fileRestaurant.write('id,link\n');
-  fileMenu.write('dishtype,subtype,dish,price,ingredients,restaurantId\n');
+  fileRestaurant.write('id|dishes|link\n');
 
   for (let id = 1; id <= 10000000; id += 1) {
```

Comment:
consider using "1e7" instead of 10000000 - easier to understand that this is 10 million